### ISSUE TITLE
Problem with DurationAxis and 100% cpu

### ISSUE THREAD

**Issue by cramsc** [amcharts/amcharts5#261]
Hi,

With some data values the chart just doesn't show up and the cpu goes to 100%. It seems to be related with DurationAxis and the second baseUnit.

This simple example does not work, but if change the data then it just works:

```

<html>
<style>
  #chartdiv {
  width: 100%;
  height: 250px;
}
</style>


<script src="//cdn.amcharts.com/lib/5/index.js"></script>
<script src="//cdn.amcharts.com/lib/5/xy.js"></script>
<script src="//cdn.amcharts.com/lib/5/themes/Animated.js"></script>
<div id="chartdiv"></div>

<script>	
	var root = am5.Root.new("chartdiv");
	
	root.setThemes([
		  am5themes_Animated.new(root)
		]);
	var chart = root.container.children.push(am5xy.XYChart.new(root, {
	  
	}));	
	
	var cursor = chart.set("cursor", am5xy.XYCursor.new(root, {
	  behavior: "none"
	}));

	var data = [
		{
			"round": "round 1",
			"seriea": 1444, //if for example i simple change to 14440 then it just works
			"serieb": 3310
		},
		{
			"round": "round 2",
			"seriea": 1415,
			"serieb": 2763
		}
	];
	
	var xAxis = chart.xAxes.push(am5xy.CategoryAxis.new(root, {
	  categoryField: "round",
	  renderer: am5xy.AxisRendererX.new(root, {}),	  
	}));
	
	var yAxis = chart.yAxes.push(am5xy.DurationAxis.new(root, {
	  baseUnit:"second",
	  renderer: am5xy.AxisRendererY.new(root, {}),
	}));

	function createSeries(name, field) {			
	  var series = chart.series.push(am5xy.LineSeries.new(root, {
		name: name,
		xAxis: xAxis,
		yAxis: yAxis,
		valueYField: field,
		categoryXField: "round",

	  }));
	  series.data.setAll(data);	
	}
	
	createSeries("Seriea", "seriea");
	createSeries("Serieb", "serieb");
	
	xAxis.data.setAll(data);
</script>
</html>


```

Thanks

**Comment by sassomedia** [comment]
Yeah, this looks like a bug. I threw this into a [pen](https://codepen.io/sassomedia/pen/podEjMG?editors=0010) based on the duration axis [example](https://www.amcharts.com/docs/v5/charts/xy-chart/axes/duration-axis/#Example). With the same numbers, it works for all baseUnits, except "second". That one freezes the tab.

**Comment by martynasma** [comment]
Fixed in [5.1.4](https://github.com/amcharts/amcharts5/releases/tag/5.1.4).

## [5.1.4] - 2022-02-11

## Added
- `affectsMinMax` added to value axis range.
- `segments` property added to `Line` class. Allows passing multiple segments instead of continous line.
- `tooltipIntervalOffset` setting added to `DateAxis`. If not set, axis' tooltip will use value of the `-tooltipLocation` (cell start timestamp by default), so that tooltip would should rounded value, like 06:00 instead of 06:30 when interval is hourly. [More info](https://www.amcharts.com/docs/v5/charts/xy-chart/axes/date-axis/#Axis_tooltip).
- `tooltipDateFormats` setting added to `DateAxis`. Allows specifying axis' tooltip date formats for time each time unit, different from `dateFormats`. NOTE: `tooltipDateFormat` will override the format if set. [More info](https://www.amcharts.com/docs/v5/charts/xy-chart/axes/date-axis/#Axis_tooltip).
- `syncWith` setting added to `XYCursor`. Set it to an array of other `XYCursor` objects to automatically position them as the cursor moves. [More info](https://www.amcharts.com/docs/v5/charts/xy-chart/cursor/#syncing-cursors).

## Changed
- Default tooltip date formats have changed slightly for `DateAxis`. Previously, it was using formats from `dateFormats`. Now it uses defaults/user values from newly-introduced `tooltipDateFormats` setting.

## Fixed
- In some cases `DurationAxis` with "second" as its base duration was going to stack overflow.
- Background on a `ValueAxis` was not showing.
- Improved performance for charts with lots of stacked series with null.
- Fixing mouse position when using `transform: scale` or `zoom` ([Issue 258](https://github.com/amcharts/amcharts5/issues/258)).
- Some touch interactions were not working in iOS 12.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
